src/burn/engine/package.cpp

```diff
@@ -371,7 +378,7 @@ extern "C" void PackageUninitialize(
     switch (pPackage->type)
     {
     case BURN_PACKAGE_TYPE_BUNDLE:
-        BundlePackageEnginePackageUninitialize(pPackage);
+        BundlePackageEnginePackageUninitialize(pPackage); // TODO: Modularization
```

Comment:
What does `Modularization` mean here? I'm guessing it doesn't mean append the Merge Module GUID to the end of primary keys. :)

Reply:
It means whatever it means for the other existing package types. You tell me what that originally meant :)

Reply:
Heh. Only guesses. 👻